dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/wim/LdapRegistry.java

```diff
@@ -69,6 +69,7 @@
     private String certificateMapperId;
     private String timestampFormat;
     private ConfigElementList<LoginProperty> loginProperties;
+    private String bindAuthMechanism;
```

Comment:
Shouldn't we have ticket cache and principal as well?

Reply:
Yes, I just didn't added them proactively.

Reply:
Added them in now.